azure-devops/cross-build-test.yml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+parameters:
+- name: hostArch
+  type: string
+  default: amd64
+- name: targetPlatform
+  type: string
+- name: vsDevCmdArch
+  type: string
+- name: buildOutputLocationVar
+  type: string
+  default: buildOutputLocation
+
+  - template: cross-build.yml

```

Comment:
This is also trigerring some errors. Is there something Im missing while calling this template?

Reply:
Is this meant to be a job or just some steps? Anyway you can't just have a free standing `- template` like that. Look at the other places template is used.

Reply:
Im looking at this example
`  - stage: Build_ARM64
    dependsOn: Code_Format
    displayName: 'Build ARM64'
    jobs:
      - template: azure-devops/cross-build-test.yml
I think the difference is that this have jobs: and displayName: over it.  But I dont think displayname matters. Shoud adding just jobs: will work?

Reply:
Yes that fixed it